ui/app/templates/vault/cluster/secrets/backend/list.hbs

```diff
@@ -1,11 +1,26 @@
+{{log backendModel}}
 <SecretListHeader
   @isCertTab={{eq tab "certs"}}
   @model={{backendModel}}
   @baseKey={{baseKey}}
   @backendCrumb={{backendCrumb}}
   @filter={{filter}}
 />
-{{#if this.noMetadataPermissions}}
+{{#if (and this.noMetadataPermissions backendModel.isV2KV)}}
```

Comment:
This is what this does; shows the create button. 
![image](https://user-images.githubusercontent.com/6618863/136090686-f396f3a1-2a94-4347-a4f4-a10d2a257062.png)